docs/openapi/aws/api-cn-CN_BE-aws.yaml

```diff
@@ -0,0 +1,350 @@
+        documentType:
+          type: string
+          description: Il tipo del documento
+          example: PN_NOTIFICATION_ATTACHMENTS
+        documentStatus:
+          type: string
+          description: Lo stato del documento
+          example: ATTACHED
+        contentType:
+          type: string
+          example: application/pdf
+        contentLength:
+          type: number
+          example: 54092
+          description: dmensione, in byte, del contenuto.
+        checksum:
+          type: string
+          description: >-
+            SHA256 o MD5 del contenuto del file. Il tipo di checksum cambia a
+            seconda del  documentType.
+        retentionUntil:
+          type: string
+          format: date-time
+          description: >-
+            data e ora fino a cui il documento sarà garantito all'interno di
+            _SafeStorage_. <br/> Può essere ritardata nel tempo, mai anticipata.
+            Formattata con timezone secondo l'rfc 3339
+
+                  
+    FileDownloadInfo:
+      type: object
+      properties:
+        url:
+          type: string
+          description: >-
+            URL preautorizzato a cui effettuare una richiesta GET per ottenere
+            il  contenuto del documento. Presente solo se il documento è pronto
+            per il download.
+        retryAfter:
+          type: number
+          description: >-
+            Stima del numero di secondi da aspettare prima che il contenuto del 
+            documento sia scaricabile.
+    ConservationEventsUpdateRequest:
+      type: object
+      properties:
+        events:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/ConservationEventUpdate'
+    ConservationEventUpdate:
+      type: object
+      required:
+        - externalId
+        - status
+        - fileKey
+        - statusDate
+      properties:
+        externalId:
+          type: string
+          description: ID del documento nel sistema di conservazione a norma
+        status:
+          type: string
+          description: Lo stato della conservazione a norma
+          example: OK
+        errorCode:
+          type: string
+          description: Codice di errore
+          example: OK
+        fileKey:
+          type: string
+          description: La chiave del documento in PN
+          example: 8F7E/9A3B/1234/AB87
+        statusDate:
+          type: string
+          format: date-time
+          description: >-
+            data e ora di aggiornamento dello stato di conservazione. <br/>
+            Formattata con timezone secondo l'rfc 3339
+        isFinal:
+          type: boolean
+          description: >-
+            True se lo stato è finale e non sono possibili ulteriore
+            aggiornamenti sullo stesso documento
+    OperationResultCodeResponse:
+      required:
+        - resultCode
+        - resultDescription
+        - clientResponseTimeStamp
```

Comment:
Manca in properties?

Reply:
@ViVeniero rimosso clientResponseTimeStamp dai required fields